pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -245,6 +245,19 @@ protected Policies getNamespacePolicies(String property, String cluster, String
         }
     }
 
+    protected LocalPolicies getNamespaceLocalPolicies(String property, String cluster, String namespace) {
```

Comment:
That's fine, but it can be quite confusing for a user given that other methods are changing the settings globally. 

I think it could either be explicitely set for a particular cluster, or for all the clusters. In order to clarify the scope. Eg: 

```json
{
 "rateLimit" : { 
     "dispatchRatePerTopicInMsg" : 1000.0,
      "dispatchRatePerTopicInBytes" : 1000000.0,
     "clusters" : {
          "small-cluster" : {
               "dispatchRatePerTopicInMsg" : 10.0,
               "dispatchRatePerTopicInBytes" : 1000.0,
          }
     } 
 }
}
```

The advantage is that you can also see all the limit in a single place.